README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Go Reference](https://pkg.go.dev/badge/github.com/rstudio/gcfgenv.svg)](https://pkg.go.dev/github.com/rstudio/gcfgenv)
 
-**This repository is a proof-of-concept meant to serve internal needs only.**
+> **NOTE**: This repository is meant to serve internal needs only.
 
```

Comment:
This was only added because we weren't sure whether we'd use it originally. I think we should just remove this line now.

```suggestion
```